rfcs/055-prismic-api/README.md

```diff
@@ -0,0 +1,249 @@
+- Though we're describing them as a separate project, the new endpoints should seamlessly fit into the rest of the wellcomecollection.org API suite from a user POV. All of the existing conventions should be followed.
+- Despite prioritising search and defaulting to the Prismic API for display, We'll still include a way to fetch a single document by ID, eg `/articles/{id}`.
+- The API's URL structure should also be consistent with what appears on the front end of the site. For example, if article on the site appears at `/articles/{id}`, the API equivalent should be at `/articles/{id}`. The same should be true for search results.
+- The new API service should be written in typescript, following patterns set by the [concepts API](../050-concepts-api/README.md) for filtering, pagination, error handling, etc.
+- We shouldn't create new IDs for exhibitions, events, stories, etc. There aren't any situations where we should need to merge content from multiple sources, so we should use the document IDs directly from Prismic.
+- The elasticsearch index mapping should represent the contract between the pipeline and the API. The API shouldn't need to know anything about the structure of the data in Prismic.
+- We won't include an endpoint for `/series`, nor will we include series nodes in the `/articles` endpoint. If an article is part of a series, we'll denormalise information from its parent onto the article itself to use for relevance and visual signposting in results.
+
+## Proposed endpoints
+
+### `/articles`
+
+#### Example response
+
+```http
+GET /articles/{id}
+{
+    "id": "",
+    "format": "",
+    "title": "",
+    "body": "", // not exposed by default, should require an ?includes=body
+    "standfirst": "",
+    "published": "",
+    "contributors": [
+        {
+            ... // reuse the Contributor model
+        },
+        ...
+    ],
+    "image": {
+        "thumbnail": {
+            ... // reuse the DigitalLocation model
+        },
+        "caption": ""
+    },
+    "partOf": [
+        {
+            ... // reuse the partOf model for series
+        }
+    ]
+}
+```
+
+```http
+GET /articles?query=foo&filters=bar&sort=baz
+{
+    "type": "ResultList",
+    "results": [
+        {
+            ... // as above
+        },
+        ...
+    ],
+    "pageSize": 100,
+    "totalResults": 1000,
+    "totalPages": 10,
+    "nextPage": "/articles?query=foo&filters=bar&sort=baz&page=2",
+}
+```
+
+#### Filters
+
+- date
+- contributor
+- series
+- contentType
+
+#### Sort
+
+- date
+- relevance
+
+### `/exhibitions`
+
+#### Example response
+
+```http
+GET /exhibitions/{id}
+{
+    "id": "",
+    "format": "",
+    "title": "",
+    "status": "",
+    "instantiations": [
+        {
+            "start": "", // ISO 8601 date only (no time)
+            "end": "", // ISO 8601 date only (no time)
+            "place": "",
+            "guide": {
+                ... // reuse the DigitalLocation model
```

Comment:
As above, I'm not sure we do reuse things like that here. We should be following how things are modelled in Prismic for the most part

Reply:
I agree for this field, but for images (which also have licensing information) I think it seems quite sensible to use DigitalLocations. Was briefly discussed and I know @alexwlchan agrees, but definitely up for debate

Reply:
I could imagine wanting to filter content that is of or has a certain type, what we currently call `interpretation` in events, e.g. `BSL video` `Audio description`. Is that what you mean by "what sorts of guides are available" @alexwlchan ?

Reply:
> Is that what you mean by "what sorts of guides are available"?

yes, exactly

Reply:
hmm, interpretation in events is generally provided live; the guides are static digital content.

Reply:
After discussing with @dannybirchall we're going with interpretations, but not the static digital content which might go into v1. 

Is there anywhere I could find example of DigitalLocations to further my understanding?